server/controllers/api.py

```diff
@@ -672,6 +672,18 @@ def can(action, resource, requester):
             return True
         return resource == requester
 
+class CourseEnrollment(Resource):
+    """ Information about all students in a course.
+    Authenticated. Permissions: >= User or admins
+    Used by: Export scripts.
+    """
+    model = models.Enrollment
+    schema = EnrollmentSchema()
+
+    @marshal_with(schema.get_fields)
+    def get(self, cid, user):
+        return {'courses': self.model.get_all_students(cid)}
```

Comment:
`EnrollmentSchema.get_fields` doesn't seem like a good fit for this. It's definitely not a list of courses. 

I'd suggest a schema that looks like 
```
{ 
  'students': fields.List,  
  'staff': fields.List, 
  'instructor': fields.List
  ... 
}
```
An easy way to generate that schema is to use a dictionary comprehension 
`all_enrollments_schema = { role: fields.List(...) for role in constants.VALID_ROLES } `